components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -2222,7 +2222,7 @@ document.observe('xwiki:dom:loaded', function()
   </object>
   <object>
     <name>PhenoTips.PatientSheetCode</name>
-    <number>8</number>
+    <number>1</number>
```

Comment:
I guess you're trying to make this one execute sooner. The whole object should be moved up in the source then.